meson.build

```diff
@@ -12,9 +12,9 @@ fwupd_minor_version = varr[1]
 fwupd_micro_version = varr[2]
 
 conf = configuration_data()
-conf.set('FWUPD_MAJOR_VERSION', fwupd_major_version)
-conf.set('FWUPD_MINOR_VERSION', fwupd_minor_version)
-conf.set('FWUPD_MICRO_VERSION', fwupd_micro_version)
+conf.set('MESON_MAJOR_VERSION', fwupd_major_version)
+conf.set('MESON_MINOR_VERSION', fwupd_minor_version)
+conf.set('MESON_MICRO_VERSION', fwupd_micro_version)
```

Comment:
That doesn't feel right; it's not the meson version, it's the fwupd project version. Maybe drop the MESON_ prefix?